pwndbg/commands/windbg.py

```diff
@@ -185,6 +185,22 @@ def bp(where):
 def bp(where):
     gdb.execute('break *%#x' % int(where))
 
+@pwndbg.commands.Command
```

Comment:
Since you're using a `Command` instead of `ParsedCommand`, you'll have to pass the additional arguments to the parser yourself.

See `pwndbg.commands.fix` for a convenient routine to do this.